src/indieweb_utils/indieauth/flask.py

```diff
@@ -113,7 +109,11 @@ def is_authenticated(
     :rtype: bool
     """
     if headers.get("Authorization") is not None:
-        access_token = headers.get("Authorization").split(" ")[-1]
+        access_token_header = headers.get("Authorization")
```

Comment:
My guess is you ran into a TypeError because `get` could return None? If we just supply a blank default it should work / let us remove an if statement? 

Thinking it through a bit further, since we check `if headers.get("Authorization") is not None` on line 111, inside that if statement should be able to use `headers["Authorization"]`, no?

If not, let's just supply a default value to get so we can reduce the number of local variables / branches.
```suggestion
        access_token_header = headers.get("Authorization", "").split(" ")[-1]
```